.github/workflows/conventional-commit-check.yml

```diff
@@ -0,0 +1,67 @@
+name: Conventional Commit Message Check
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - ready_for_review
```

Comment:
do we also need to include

```
synchronize
```

Reply:
We only check for PR title. I don't see the need of running this check each time the branch has been updated.